scripts/get-opt-relay-cost.sh

```diff
@@ -4,34 +4,56 @@ trap 'kill $(jobs -p) 2>/dev/null' EXIT
 
 [[ "$(cast chain --rpc-url="$ETH_RPC_URL")" == "ethlive" ]] || { echo "Please set a Mainnet ETH_RPC_URL"; exit 1; }
 
-OPTIMISM_GOERLI_RPC_URL='https://mainnet.optimism.io'
+OPTIMISM_MAINNET_RPC_URL='https://mainnet.optimism.io'
 
 L2_SPELL='0x9495632F53Cc16324d2FcFCdD4EB59fb88dDab12'
```

Comment:
nit: wondering if this bit could be passed as input and the script added also in the Makefile, following previous patterns.